docs/api/doc-block-source.md

```diff
@@ -102,7 +102,16 @@ Type: `boolean | 'dedent' | BuiltInParserName`
 
 Default: `parameters.docs.source.format` or `true`
 
-Specifies the formatting used on source code. Supports all valid [prettier parser names](https://prettier.io/docs/en/configuration.html#setting-the-parserdocsenoptionshtmlparser-option).
+Specifies the formatting used on source code. `true` and `'dedent'` both does the same thing, removing any extraonous indentation. Supports all valid [prettier parser names](https://prettier.io/docs/en/configuration.html#setting-the-parserdocsenoptionshtmlparser-option).
```

Comment:
@JReinhold couldn't we add a small tweak to the second sentence, turning it into: " Both `true` and `'dedent'` have the same effect of removing any extraneous indentation.". Let me know and we'll go from there.